client/state/sharing/publicize/actions.js

```diff
@@ -71,6 +75,44 @@ export function fetchConnections( siteId ) {
 }
 
 /**
+ * Triggers a network request to request a Publicize connection for the
+ * specified site ID.
+ *
+ * @param  {Number} siteId       Site ID
+ * @param  {Number} connectionId ID of the connection to be fetched.
+ * @param  {String} label        Name of the service to be fetched.
+ * @return {Function}            Action thunk
+ */
+export function fetchConnection( siteId, { ID: connectionId, label } ) {
```

Comment:
> Do we not get a label from the error response?

No, if there's an error it's most likely due to a missing connection or missing creds. But that's fine, we can display a generic error message